experimental/dds/tree2/src/feature-libraries/editable-tree-2/editableTreeTypes.ts

```diff
@@ -190,6 +190,18 @@ export interface TreeField extends Tree<FieldSchema>, Iterable<TreeNode> {
  * @alpha
  */
 export interface MapNode<TSchema extends MapSchema> extends TreeNode {
+	/**
+	 * Checks whether a value exists for the given key.
+	 * @param key - which map entry to look up.
+	 *
+	 * @remarks
+	 * All fields under a map implicitly exist, but `has` will only return true if there are one or more nodes present in the given field.
```

Comment:
It might be worth elaborating more on the statement "All fields under a map implicitly exist" - it isn't immediately clear to me what this means.

Reply:
It's a bit of an unintuitive concept, I agree. For now, I'm just adding it to be consistent with `get()`. When we decide how to better express this concept (which we will need to, it's a known problem), then we can revise them all at once. You'll see similar comments on methods in the related PRs.